static/js/publisher/market/market.js

```diff
@@ -320,6 +322,34 @@ function initForm(config, initialState, errors) {
     validateInput(event.target);
     updateFormState();
   });
+
+  // Prefix contact and website fields on blur if the user doesn't provide the protocol
+  function prefixInput(input) {
+    if (['website', 'contact'].includes(input.name)) {
+      if (
+        validation[input.name].isValid &&
+        input.value.length > 0 &&
+        !input.value.includes('http') &&
+        !input.value.includes('mailto')
+      ){
+        if (input.name === 'website') {
+          // Default to http as most sites redirect to https if available
+          input.value = `https://${input.value}`;
+        } else if (input.name === 'contact' && input.value.includes('@') && !input.value.includes('mailto')) {
```

Comment:
Maybe instead of just checking that it includes '@' we could use `MAILTO_REGEXP` to make sure it's a mail?